cmd/controller/cmd/webhook.go

```diff
@@ -0,0 +1,158 @@
+package cmd
```

Comment:
package documentation?

Reply:
I added docs to each command... I don't know what would make sense to put in the commands package docs... please review